### PR TITLE
add output_type parameter

### LINKED ISSUE #121: MlcpTask does not support the output_type parameter
The MlcpTask is missing a parameter for output_type.  I know I can pass additional arguments in an arg tag, but it would be useful to have it be a property  configured for the task.

### PR DESCRIPTION
**Pull request by tzejohn**
This patch fixes #121 .

### CODE REVIEW

**Comment by rjrudin** [comment]
Want to add any other parameters while you're at it? =)

There are a number of others to add. I think there's a fancy way in Groovy to be able to use e.g. the MlcpBean class in my main mlcp-util project and somehow expose all of its properties as task arguments, as opposed to having to list each one out on the task. But listing them out on the task is fine too, not a big deal.

**Comment by tzejohn** [comment]
I’ll try to implement based on the MlcpBean.

**Comment by tzejohn** [comment]
Ok.  Not sure if this is what you are looking for, but I switched MlcpTask to use MlcpBean as a @Delegate.  And then kind of kept the arg building of the original task.

**Comment by rjrudin** [comment]
Nice, I'll try this out by the end of the week and get it into a bugfix release - 2.3.3

**Comment by rjrudin** [comment]
This looks good, but I'm going to make a change to mlcp-util so that it doesn't depend on mlcp, as I don't want ml-gradle to depend on mlcp. I'm going to change mlcp-util to just collect and return arguments. 

So I'll merge this locally and then integrate in a new version of mlcp-util.

File: src/main/groovy/com/marklogic/gradle/task/MlcpTask.groovy [hunk 1]
```diff
@@ -16,32 +20,12 @@ import com.marklogic.appdeployer.AppConfig
...
-    String output_permissions
-    String transform_module
-    String transform_namespace
-    String transform_param
-    String thread_count
-    
+    @Delegate MlcpBean mlcpBean = new MlcpBean();
+
+    public Logger getLogger() {
+        return Logging.getLogger(MlcpTask.class)
```
Comment (rjrudin) [inline src/main/groovy/com/marklogic/gradle/task/MlcpTask.groovy]: @tzejohn What's the result of overriding this?

Comment (tzejohn) [inline src/main/groovy/com/marklogic/gradle/task/MlcpTask.groovy]: If I did not override, the method signature of getLogger() from the Delegate MlcpBean conflicted with the signature for getLogger() in MlcpTask.

**Comment by rjrudin** [comment]
Going to merge this into master, then upgrade to mlcp-util 0.3.0

**Comment by rjrudin** [comment]
@tzejohn This is now available in version 2.3.3 of ml-gradle

**Pull request merged**